app/tests/spec/models/account.js

```diff
@@ -298,6 +298,47 @@ define(function (require, exports, module) {
       });
     });
 
+    describe('signOut', function () {
+      beforeEach(function () {
+        sinon.stub(fxaClient, 'signOut', function () {
+          return p();
+        });
+
+        account.set('sessionToken', SESSION_TOKEN);
+
+        return account.signOut();
+      });
+
+      it('calls the correct fxaClient method', function () {
+        assert.isTrue(fxaClient.signOut.calledWith(SESSION_TOKEN));
```

Comment:
Not sure if this makes me a paranoid weirdo, but I always like to assert the value of `.callCount` so that I know there is only one call to the method I'm interested in. Am I a paranoid weirdo?